.github/workflows/validate.yml

```diff
@@ -0,0 +1,28 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  release:
+    types:
+      - created
+name: Validate
+jobs:
+  Go:
+    strategy:
+      matrix:
+        go-version: [1.13.x, 1.14.x, 1.15.x]
+        os: [ubuntu-latest]
```

Comment:
In the docker file we build the image on ubuntu 18.04 ... should we mirror that perhaps?

Reply:
I can change it to ubuntu-18.04, or have both ubuntu-18.04 and ubuntu-20.04.

At the moment ubuntu-latest is the same as ubuntu-18.04: https://docs.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners#supported-runners-and-hardware-resources

Reply:
> I can change it to ubuntu-18.04, or have both ubuntu-18.04 and ubuntu-20.04.

How about just ubuntu-18.04 and we'll keep it updated along with the docker file. Don't want this matrix to get too large.